dev/manual_tests/overlay_geometry.dart

```diff
@@ -163,8 +163,9 @@ class OverlayGeometryAppState extends State<OverlayGeometryApp> {
     });
   }
 
-  void handleTapUp(GlobalKey target, Point globalPosition) {
+  void handleTapUp(Key t, Point globalPosition) {
```

Comment:
What do you mean by just change the signature here? Do you mean my version or your version?
I assume you mean @checked GlobalKey. correct?
Note that with @checked GlobalKey, you will still have a static error at line 190 ("onTapUp: handleTapUp"). You could add a cast there as well if you wanted and that cast would pass at runtime if @checked GlobalKey was used.